server/src/bin/dump-schema.rs

```diff
@@ -0,0 +1,37 @@
+use anyhow::Result;
+use juniper::{introspect, IntrospectionFormat};
+use stacks_exchange::{
+    db::make_pool,
+    graphql::{make_schema, Context},
+    settings::Settings,
+};
+use std::path::PathBuf;
+use structopt::StructOpt;
+
+#[derive(Debug, StructOpt)]
+#[structopt(name = "dump_schema")]
+struct Opt {
+    #[structopt(parse(from_os_str), short, long)]
+    conf: Option<PathBuf>,
+}
+
+fn main() -> Result<()> {
+    dotenv::dotenv().ok();
```

Comment:
dotenv() loads the `.env` file into the environment, and returns a Result<PathBuf>. The ok() call converts it into an Option<PathBuf> to suppress a warning about an unused environment variable.

But come to think of it there's no reason to suppress that error - an issue at this stage will almost certainly lead to an error a few loc later, so might as well change it to `dotenv::dotenv()?` to raise the issue sooner

Reply:
Oh wait, it's not necessarily an error - it's perfectly fine to pass the required environment variables directly instead of using `.env`. So I'm leaving this as is